.github/workflows/integration-tests.yml

```diff
@@ -129,6 +132,13 @@ jobs:
         run: |
           poetry run python tests/integration/integration_tests.py
 
+      - name: Run integration tests - Anthropic
+        env:
+          CODEGATE_PROVIDERS: "anthropic"
+          ENV_ANTHROPIC_KEY: ${{ secrets.anthropic-key }}
```

Comment:
we might want to do a matrix in the future, but this is good enough

Reply:
I was thinking about that too, but I wanted to get everything running first and then do that as a follow up 👍 That way they can run in parallel too